### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by foxted**
Quick translations & sync check.

### CODE REVIEW

**Comment by jbrooksuk** [comment]
Thanks!

File: app/lang/fr/setup.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
     'title'             => 'Installer Cachet',
     'service_details'   => 'Details du service',
     'status_page_setup' => 'Installation de la page de statut',
-    'show_support'      => 'Show support for Cachet? <small>Places a link in the footer linking to Cachet.</small>',
+    'show_support'      => 'Montrer votre support pour Cachet? <small>Affiche un lien vers Cachet dans le pied de page.</small>',
     'admin_account'     => 'Compte administrateur',
     'complete_setup'    => 'Terminer l\'installation',
     'completed'         => 'Cachet a été configuré avec succès !',
```
Comment (GrahamCampbell) [inline app/lang/fr/setup.php]: shouldn't be a space before the !

Comment (willdurand) [inline app/lang/fr/setup.php]: Before the `!` nope, but before `?` yes, for sure!

Comment (jbrooksuk) [inline app/lang/fr/setup.php]: Unless that's how the French do grammar ;)

Comment (GrahamCampbell) [inline app/lang/fr/setup.php]: there shouldn't be spaces anywhere.

Comment (willdurand) [inline app/lang/fr/setup.php]: In French, there are spaces before any punctuation :)

Comment (GrahamCampbell) [inline app/lang/fr/setup.php]: Hmmm, it seems to be a stylistic choice. It does seem to be preferred in that language. I think we shouldn't do it though. It spoils the consistency.

**Comment by foxted** [comment]
True, there should be a space before the ?, missed that!

File: app/lang/fr/setup.php (hunk 1 above)
Comment (willdurand) [inline app/lang/fr/setup.php]: Allow me to disagree, it is not a stylistic choice. See #313

Comment (GrahamCampbell) [inline app/lang/fr/setup.php]: Ok then. I'm far from an expert. :)

Comment (Crash--) [inline app/lang/fr/setup.php]: In French, we have to add a space before every "double ponctuation" (i.e. : ; ! ?) it's the rule, not a stylistic choice.

Comment (jbrooksuk) [inline app/lang/fr/setup.php]: @GrahamCampbell if that's how French write, then we should do it. It should be a natural translation.

**Pull request merged**